lib/precompiled/06-ecadd.js

```diff
@@ -20,17 +20,15 @@ module.exports = function (opts) {
     return results
   }
 
-  try {
-    let returnData = ecAddPrecompile(inputHexStr)
-    results.return = Buffer.from(returnData, 'hex')
-    results.exception = 1
-  } catch (e) {
-    console.log('exception in ecAdd precompile is expected. ignore previous panic...')
-    // console.log(e)
+  let returnData = ecAddPrecompile(inputHexStr)
+  if (returnData === "") {
```

Comment:
What is type being returned? Buffer? String? I think it should be safe to use `returnData.length === 0` or even `returnData.length !== 32` as we expect a 256bit EC point (or is it a 65 bytes for type/sign, x, y?)

Reply:
Wouldn't it make sense here to instead of returning just the empty string returning somthing like "RUSTBNERROR: Something happened!" and then check for (length +) the ``RUSTBNERROR`` substring? Then this would be more explicit / readable in the code and there would be more flexibility expanding the rustbn.js library to return more meaningful output.

Reply:
Thanks @axic and @holgerd77 for your feedback, I've changed the ethereumjs-vm code to check the rustbn.js output length, for ecadd and ecmul the output should be 128, and for ecpairing the output length should be 64.